### ISSUE TITLE
[Question] Can't see display historical data on dashboard

### ISSUE THREAD

**Issue by econeale** [thingsboard/thingsboard#4796]
**Component**

 * UI

**Description**
I am trying to display historical data on a dashboard from a single device. However, I can't seem to get it to display anything! Help would be most appreciated :) 

So far I have verified that: 
1. My device is registered and reporting data
![Screen Shot 2021-06-25 at 9 28 13 AM (3)](https://user-images.githubusercontent.com/82894027/123433618-a7641a00-d599-11eb-8b46-ddcdf8fb1895.png)

2. My rule engine is processing messages and reporting telemetry data as telemetry data with a TTL of 2592000
![Screen Shot 2021-06-25 at 9 27 15 AM (3)](https://user-images.githubusercontent.com/82894027/123433699-bcd94400-d599-11eb-83e8-f641a254f709.png)

3. My Alias has a single device (auto populated with the name)
![Screen Shot 2021-06-25 at 9 32 40 AM (3)](https://user-images.githubusercontent.com/82894027/123434195-37a25f00-d59a-11eb-9309-81ed0c9b96b4.png)

4. My data source has the correct alias and no filter. 
![Screen Shot 2021-06-25 at 9 32 50 AM (3)](https://user-images.githubusercontent.com/82894027/123434544-9962c900-d59a-11eb-88c0-31c53175a001.png)

5. I thought maybe the telemetry value was being interpreted as a string so I tried dumping it to a table with no luck :(
![Screen Shot 2021-06-25 at 9 26 39 AM (3)](https://user-images.githubusercontent.com/82894027/123434038-0d50a180-d59a-11eb-9e56-4aab752be754.png)

**Environment**

 * OS:  N/A
 * ThingsBoard: Cloud
 * Browser: Chrome (latest)

**Comment by LeoMorgan113** [comment]
Hi @econeale,

Please, check if your device profile has the correct Rule Chain.
Also you can check thingsboard.log file, where may be written the cause of the problem.

**Comment by econeale** [comment]
Hi @LeoMorgan113  - thanks for the reply! It looks like the Rule Chain setting in the device profile wasn't set to anything. When I set it to the root rule chain, it didn't seem to make a difference. However, when I deleted/reprovisioned the device it started working as expected. Not really sure what that means but it's doing what I want now so I'm satisfied! 

Thanks! I'm going to go ahead and close this question.
